specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2025-07-01-preview/PreviewAlertRule.json

```diff
@@ -0,0 +1,510 @@
+        }
+      },
+      "description": "A rule preview result containing a dimension combination and an array of evaluations."
+    },
+    "Evaluation": {
+      "type": "object",
+      "properties": {
+        "evaluationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time when the evaluation was performed."
+        },
+        "alertState": {
+          "type": "string",
+          "enum": [
+            "NoAlert",
+            "Fired",
+            "Firing",
+            "Resolving",
+            "Resolved"
+          ],
+          "x-ms-enum": {
+            "name": "AlertState",
+            "modelAsString": true
+          },
+          "description": "The state of the alert after the evaluation."
+        },
+        "evaluatedPeriods": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EvaluatedPeriod"
+          },
+          "description": "An array of evaluated periods."
+        },
+        "thresholdMet": {
+          "type": "boolean",
+          "description": "Indicates whether the threshold was met for this evaluation."
+        }
+      },
+      "description": "An evaluation of the alert rule."
+    },
+    "EvaluatedPeriod": {
+      "type": "object",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp of the evaluated period."
+        },
+        "highThreshold": {
+          "type": "number",
+          "format": "double",
+          "description": "The high threshold value for the evaluation period."
+        },
+        "lowThreshold": {
+          "type": "number",
+          "format": "double",
+          "description": "The low threshold value for the evaluation period."
+        },
+        "metricValue": {
+          "type": "number",
+          "format": "double",
+          "description": "The metric value for the evaluation period."
+        },
+        "thresholdMet": {
+          "type": "boolean",
+          "description": "Indicates whether the threshold was met during the evaluation period."
+        }
+      },
+      "description": "Details of the evaluated period."
+    },
+    "DimensionNameAndValue": {
+      "type": "object",
+      "description": "Dimension name and value",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the dimension"
+        },
+        "value": {
+          "type": "string",
+          "description": "Value of the dimension"
+        }
+      },
+      "required": [
+        "name",
+        "value"
+      ]
+    },
+    "PreviewAlertRuleRequest": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The location of the rule resource."
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "timespan": {
```

Comment:
Is it intended to rename 'lookbackPeriod' compared to iteration 1? I assume iteration 1 is based on the previous api version?